### PR TITLE
Index benchmark metadata

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description

Index benchmark metadata leveraging the run_benchmark helper. Some benchmarks such as router-perf does not use that wrapper, we'll have to implement this functionallity separately.
Also renaming the start_date to the common timestamp field

### CODE REVIEW

**Comment by mkarg75** [comment]
LGTM!

**Comment by rsevilla87** [comment]
> hehe, that's why I asked you :D

Sure, I was going nuts cuz I couldn't find any document :D

File: utils/common.sh [hunk 1]
```diff
@@ -177,15 +178,16 @@ local METADATA=$(cat << EOF
 "total_nodes":"${TOTAL_NODES}",
 "sdn_type":"${SDN_TYPE}",
 "benchmark":"${BENCHMARK}",
-"start_date":"${START_DATE}",
+"timestamp":"${START_DATE}",
```
Comment (Harshith-umesh) [inline utils/common.sh:181]: why are we changing this?

File: workloads/kube-burner/common.sh [hunk 2]
```diff
@@ -142,6 +136,9 @@ check_running_benchmarks() {
 }
 
 cleanup() {
+  log "Cleaning up benchmark"
+  kubectl delete -f ${TMPCR}
+  kubectl delete configmap -n benchmark-operator kube-burner-cfg-${UUID}
   if ! oc delete ns -l kube-burner-uuid=${UUID} --grace-period=600 --timeout=30m; then
```
Comment (Harshith-umesh) [inline workloads/kube-burner/common.sh:142]: if we're doing this , should we close this pr?

Comment (rsevilla87) [reply workloads/kube-burner/common.sh:142]: yep

File: utils/common.sh (hunk 1 above)
Comment (rsevilla87) [reply utils/common.sh:181]: We generally use timestamp as date field in our indexes and grafana is only able to handle a time field (Except for uperf which is uperf_ts, that's something we'll have to change to stantarize)

Comment (Harshith-umesh) [reply utils/common.sh:181]: ohh right yeah, this was added recently

**Comment by Harshith-umesh** [comment]
LGTM

**Pull request merged**